src/Shared/Recruit.Vacancies.Client/Domain/Entities/ReportSummary.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public class ReportSummary
+    {
+        public Guid Id { get; set; }
+        public ReportOwner Owner { get; set; }
+        public ReportStatus Status { get; set; }
+        public ReportType ReportType { get; set; }
+        public List<ReportParameter> Parameters { get; set; }
```

Comment:
Could this not be a dictionary of string, object and avoid duplicates if there are more parameters in future

Reply:
I thought about this at the time and thought there may be reason why you would need duplicates. I.e. passing in multiple UKPRNs. However I guess there is no reason why you couldn't just have a single parameter called `UKPRNs` whose value is a list. On balance I think having a dictionary may prevent silly mistakes from happening so will change. 👍